packages/dev/core/src/Materials/Textures/videoTexture.ts

```diff
@@ -438,7 +449,7 @@ export class VideoTexture extends Texture {
 
         return new Promise<VideoTexture>((resolve) => {
             const onPlaying = () => {
-                const videoTexture = new VideoTexture("video", video, scene, true, invertY);
+                const videoTexture = new VideoTexture("video", video, scene, true, invertY, undefined, undefined, undefined, Constants.TEXTUREFORMAT_RGB);

```

Comment:
It is not possible that a stream has an alpha channel? Shouldn't we pass the format to `CreateFromStreamAsync`?